src/python/bin/aqueduct

```diff
@@ -297,12 +331,25 @@ if __name__ == "__main__":
     if not os.path.isdir(base_directory):
         os.mkdir(base_directory)
     
-    if args.command == "server":
-        server(args.expose)
+    if args.command == "start":
+        try:
+            server_popen = server(args.expose)
+            ui_popen = ui(args.expose)
+            print(generate_welcome_message(args.expose))
```

Comment:
This will get printed out *after* the logs for the server setup, correct?

Reply:
So this is a little tricky: although the `print` is below the server and ui process start lines, since these processes are launched asynchronously, in practice I always see the welcome message getting printed at the very beginning, followed by some server and UI logs.

Reply:
Okay, can we insert a small waiting period here? 2-3 seconds?